src/components/forms/FamilyForm.tsx

```diff
@@ -398,12 +437,47 @@ export const FamilyForm = ({ family: loadedFamily }: TProps) => {
       {canLoadForm && (
         <>
           <form onSubmit={handleSubmit(onSubmit)}>
+            {isLeavingModalOpen && (
+              <Modal
+                isOpen={isLeavingModalOpen}
+                onClose={() => setIsLeavingModalOpen(false)}
+              >
+                <ModalOverlay />
```

Comment:
This file is getting larger now and is already quite large, can we please look at creating a new folder under `/components/modals` for modals to live, passing in any relevant show/hide functions, and state values as props?